reference/5.1/Microsoft.PowerShell.Core/About/about_Scopes.md

```diff
@@ -640,50 +638,51 @@ In Sample.ps1:
 "The value of `$Ptest is $global:Ptest."
 ```
 
-The `$ptest` variable is not visible in the script scope, the output is
-empty.
+The `$ptest` variable isn't visible in the script scope, the output is empty.
 
 ```powershell
 "The value of $Ptest is ."
 "The value of $Ptest is ."
 ```
 
-### Example 5: Using a Local Variable in a Remote Command
+### Example 5: Using a local variable in a remote command
 
-For variables in a remote command created in the local session, use the `Using`
+For variables in a remote command created in the local session, use the `using`
 scope modifier. PowerShell assumes that the variables in remote commands were
 created in the remote session.
 
 The syntax is:
 
 ```
-$Using:<VariableName>
+$using:<VariableName>
 ```
 
 For example, the following commands create a `$Cred` variable in the local
 session and then use the `$Cred` variable in a remote command:
 
 ```powershell
 $Cred = Get-Credential
-Invoke-Command $s {Remove-Item .\Test*.ps1 -Credential $Using:Cred}
+Invoke-Command $s {Remove-Item .\Test*.ps1 -Credential $using:Cred}
 ```
 
-The Using scope was introduced in PowerShell 3.0. In PowerShell 2.0, to
-indicate that a variable was created in the local session, use the following
-command format.
-
-```powershell
-$Cred = Get-Credential
-Invoke-Command $s {
-  param($c)
-  Remove-Item .\Test*.ps1 -Credential $c
-} -ArgumentList $Cred
-```
+The `using` scope modifier was introduced in PowerShell 3.0.
 
 ## See also
 
-- [about_Variables](about_Variables.md)
-- [about_Environment_Variables](about_Environment_Variables.md)
-- [about_Functions](about_Functions.md)
-- [about_Script_Blocks](about_Script_Blocks.md)
-- [Start-ThreadJob](/powershell/module/ThreadJob/Start-ThreadJob)
+- [about_Variables][08]
+- [about_Environment_Variables][03]
+- [about_Functions][04]
+- [about_Script_Blocks][07]
+- [Start-ThreadJob][01]
+
+<!-- link references -->
+[01]: /powershell/module/ThreadJob/Start-ThreadJob
+[02]: #private-option
+[03]: about_Environment_Variables.md
+[04]: about_Functions.md
+[05]: about_operators.md
```

Comment:
this should be changed to  `about_Operators.md`. At least, this is required when browsing the code in GitHub, which is case-sensitive